### ISSUE TITLE
Contrast of built-in radiotable themes

### ISSUE THREAD

**Issue by TomaszGasior** [TomaszGasior/RadioLista-v3#57]
It seems to be some built-in  themes for radiotables have poor contrast. It's needed to investigate this problem. I will use https://wave.webaim.org/ for this task.

**Comment by TomaszGasior** [comment]
Also, it's needed to investigate contrast ratio of built-in themes on mobile version, where background image is not visible. 

Also, RDS field background color (focused too) should be verified.
